MAUI/TreeView/getting-started.md

```diff
@@ -0,0 +1,860 @@
+---
+layout: post
+title: Getting Started with .NET MAUI TreeView Control | Syncfusion
+description: Learn here about getting started with the Syncfusion .NET MAUI TreeView (SfTreeView) control, its elements and more.
+platform: MAUI
+control: SfTreeView
+documentation: ug
+---
+
+# Getting Started with .NET MAUI TreeView (SfTreeView)
+
+This section provides a quick overview for getting started with the TreeView with the .NET Maui TreeView(SfTreeView) for Maui. Walk through the entire process of creating real world of this control.
+
+
+## Creating an application using the .NET MAUI TreeView
+
+ 1. Create a new .NET MAUI application in Visual Studio.
+ 2. The Syncfusion .NET MAUI components are available on nuget.org. To add SfTreeView to your project, open the NuGet package manager in Visual Studio, search for Syncfusion.Maui.TreeView and then install it.
```

Comment:
add link for "nuget.org" (refer our other controls UG)

Reply:
Added link for nuget.org and highlighted the package name